src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
Are we planning on shipping the slow span assembly in the future for some runtime other than for .NET Core 2.0? What I'm not following is that we're building it currently for netstandard1.0 - are we planning on shipping the assembly again for that? If not, why build it? If we are planning to ship it for other runtimes like UAP or CoreRT then I would imagine we would build for those configurations, not for netstandard1.0.